docs/02-deployment-guide.md

```diff
@@ -281,31 +343,38 @@ SNS topic used by other stacks to send errors and finally the
 to the configuration bucket.
 
 That leaves the following stacks as optional:
+
 * `CompactionStack` - for running compactions (in practice this is essential)
 * `GarbageCollectorStack` - for running garbage collection (in practice this is essential)
 * `IngestStack` - for ingesting files using the "standard" ingest method
 * `PartitionSplittingStack` - for splitting partitions when they get too large
 * `QueryStack` - for handling queries
 * `EmrBulkImportStack` - for running BulkImport jobs using Spark running on an EMR cluster that is created on demand
-* `PersistentEmrBulkImportStack` - for running BulkImport jobs using Spark running on a persistent EMR cluster, i.e. one that
-is always running (and therefore always costing money). By default, this uses EMR's managed scaling to scale up and down on
-demand.
-* `DashboardStack` - for creating Cloudwatch metrics showing statistics such as the number of records in a table over time
+* `PersistentEmrBulkImportStack` - for running BulkImport jobs using Spark running on a persistent EMR cluster, i.e. one
+  that
+  is always running (and therefore always costing money). By default, this uses EMR's managed scaling to scale up and
+  down on
+  demand.
```

Comment:
Not sure why these have changed to be on separate lines?

Reply:
IDE autoformatting, I'll fix it.

Reply:
Done